go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -542,6 +524,58 @@ func (bsnc *BaseSecondaryNetworkController) removePodForSecondaryNetwork(pod *ka
 	return nil
 }
 
+// hasIPAMClaim determines whether a pod's IPAM is being handled by IPAMClaim CR.
+// pod passed should already be validated as having a network connection to nadName
+func (bsnc *BaseSecondaryNetworkController) hasIPAMClaim(pod *kapi.Pod, nadNamespacedName string) (bool, error) {
```

Comment:
yes. `nadName` is in `<ns>/<metadata.name>` format for **all** secondary networks.

Keep in mind it'll be `default` for the cluster default network! But I don't think this **cannot** explode since we only invoke this method for network managed by the `BaseSecondaryNetworkController`. Which I assume can't manage the default network NAD.